programs/survival/simply_harvest.sc

```diff
@@ -16,6 +16,9 @@ global_seeds = {
     'nether_wart' -> 'nether_wart'
 };
 
+// variable indicating that harvesting occurred
+global_harvesting = false;
+
```

Comment:
Where in the code do we actually query this variable tho?

Reply:
In __on_player_placing_block (ran after the main event handler if offhand) and in the 0-scheduled task (where it's just set back to false).

Reply:
Mah bad, didn't read properly